src/pages/workspaces/workspace/WorkspaceMenu.test.ts

```diff
@@ -460,7 +469,7 @@ describe('DynamicWorkspaceMenuContent fetches specific workspace details', () =>
     // Assert
     const menuItem = screen.getByText('Clone');
     await user.click(menuItem);
-    expect(onClone).toBeCalledWith([protectedDataPolicy], undefined);
+    expect(onClone).toBeCalledWith([protectedDataPolicy], undefined, descriptionText);
```

Comment:
Please update the descriptions of the modified test cases to include that they are testing the description override as well.